orttraining/orttraining/python/training/__init__.py

```diff
@@ -4,3 +4,4 @@
 #--------------------------------------------------------------------------
 from onnxruntime.capi._pybind_state import TrainingParameters
 from onnxruntime.capi.training.training_session import TrainingSession
+from . import amp
```

Comment:
nip: what is amp standing for? will it be confused with apex.amp?

Reply:
automatic mixed precision and the name matches apex and pytorch amp on purpose, so that users know that onnxrutnime.amp.loss_scaler is similar to apex.amp.loss_scaler, for example